src/wiki/getting-started/install-of-alternatives.md

```diff
@@ -7,70 +7,104 @@ eleventyNavigation:
 
 # Why bother?
 
-Once upon a time, OptiFine was *the* way, to make your game better, faster, and stronger. Things have since changed.
+Once upon a time, OptiFine was *the* way to make your game better, faster, and stronger through various useful features and optimizations. Things have since changed.
 
-## <img src="https://raw.githubusercontent.com/FabricMC/community/main/media/unascribed/png/fabric.png" height="20"><img src="https://raw.githubusercontent.com/QuiltMC/art/master/brand/svg/quilt_logo_dark.svg" height="20"> Fabric/Quilt
+## Detailed explanation
 
-**NOTE:** Most Fabric mods require the [Fabric API](https://modrinth.com/mod/fabric-api) mod in order to work, while most Quilt mods require [Quilted Fabric API](https://modrinth.com/mod/qsl) to work.
+For a very long time, OptiFine was an essential mod that many enjoyed using as it offers important benefits such as performance improvements, extra quality of life features, and more. However, it also has it's downsides:
 
-### <img src="https://cdn.modrinth.com/data/AANobbMI/icon.png" height="20"> Sodium
+- OptiFine is closed-source. This means that no one can easily inspect it's code, and it is much more difficult for developers to make their mods compatible with OptiFine - since they can't see what could be conflicting between their mod and OptiFine.
```

Comment:
```suggestion
- OptiFine is closed-source. This means that no one can easily inspect its code, and it is much more difficult for developers to make their mods compatible with OptiFine - since they can't see what could be conflicting between their mod and OptiFine.
```